iocore/net/SSLUtils.cc

```diff
@@ -2149,6 +2112,7 @@ ssl_callback_session_ticket(SSL *ssl, unsigned char *keyname, unsigned char *iv,
                             int enc)
 {
   SSLCertificateConfig::scoped_config lookup;
+  SSLConfig::scoped_config params;
```

Comment:
That isn't safe.  We need to acquire the reference to the SSLConfig before we potentially reference default_global_keyblock and we need to hold the reference through any lines that might potentially reference the keyblock value.  If we move the reference into the block at actually pulls the default_global_keyblock, we leak an unprotected reference to that keyblock.